codex-ui/src/vue/input/Input.vue

```diff
@@ -29,10 +27,17 @@ const props = withDefaults(
     disabled?: boolean;
   }>(),
   {
+    value: '',
```

Comment:
Why value is empty? Why put it in here at all if it's optional?

Reply:
It don't make a big difference, since this prop is optional. 
In TypeScript's `lib.dom.d.ts` HTMLInputElement has a `value` prop which is `string`. So it can't be `undefined`